subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -1212,7 +1212,10 @@ static void isr_done(void *param)
 #endif /* CONFIG_BT_CTLR_ADV_INDICATION */
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT) || defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
-	ull_done_extra_type_set(EVENT_DONE_EXTRA_TYPE_ADV);
+	/* If no auxiliary PDUs scheduled, generate primary radio event done */
+	if (!lll->aux) {
+		ull_done_extra_type_set(EVENT_DONE_EXTRA_TYPE_ADV);
+	}
```

Comment:
@mtpr-ot I had a merge conflict, and I think this `if` clause I added in this PR needs your help to rework requirements of JIT scheduling. This PR tries to generate primary done event for primary plus auxiliary PDU transmission tuple. Please provide a correct port to ensure JIT scheduling requirement is not broken.

Reply:
I am putting this PR back to review, so that issue at hand is fixed upstream. The required port to handle JIT scheduling can be sent subsequently.